hack/fetch-ext-bins.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+# 
+# Copyright 2021.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Enable tracing in this script off by setting the TRACE variable in your
+# environment to any value:
+#
+# $ TRACE=1 test.sh
+TRACE=${TRACE:-""}
+if [ -n "$TRACE" ]; then
+  set -x
+fi
+
+# TODO: update to k8s 1.17
+k8s_version=1.16.4
```

Comment:
Where did you get this from? We will want to update this to a newer version

Reply:
[https://github.com/openshift/cluster-api-provider-aws/blob/master/hack/fetch-ext-bins.sh#L29](AWS)
[https://github.com/openshift/cluster-api-provider-gcp/blob/master/hack/fetch-ext-bins.sh#L29](GCP)

Reply:
Thanks, they both need an update, we definitely have more up to date versions, will try and dig one out